src/graphql/resolvers.ts

```diff
@@ -56,6 +56,34 @@ export const resolvers = {
             });
             return newUser; 
         },
+        login: async (_, args) => {
+            const user = await prisma.user.findUnique({
+                where: {
+                    email: args.data.email,
+                },
+            }); 
+
+            if (!user) {
+                throw new CustomError(401, 'Credenciais invalidas.');
+            }
+
+            const checkPassword = await bcrypt.compare(args.data.password, user.password);
+
+            if (!checkPassword) {
+                throw new CustomError(401, 'Credenciais invalidas.');
+            } else {
+                return {
+                    user: {
+                        id: user.id, 
+                        name: user.name, 
+                        email: user.email, 
+                        birthDate: user.birthDate,
+                    }, 
+                    token: '',
+                }
+            }
```

Comment:
```suggestion
            if (!checkPassword) {
                throw new CustomError(401, 'Credenciais invalidas.');
            }
                return {
                    user: {
                        id: user.id, 
                        name: user.name, 
                        email: user.email, 
                        birthDate: user.birthDate,
                    }, 
                    token: '',
                }
            ```

ja que ja ta dando o early throw, poderia omitir o else e seguir a vida, isso ajuda na leitura para o caso de ter mais lógica depois (e vai ter, na hora de montar o token)